src/components/Judging/Submission.js

```diff
@@ -50,28 +52,33 @@ const FeedbackItem = styled(Card)`
   &:last-child {
     margin-bottom: 0;
   }
+  ${p => p.reported && `border: 1px solid ${p.theme.colors.warning};`}
 `
 
 const Column = styled.div`
   flex: ${p => p.width || 1};
 `
 
-const FeedbackCard = ({ feedback }) => {
+const StyledButton = styled(Button)`
+  float: right;
+`
+
+const FeedbackCard = ({ feedback, reportCallback }) => {
   return (
     <>
-      {feedback.map((item, i) => {
+      {Object.entries(feedback).map(([i, item]) => {
```

Comment:
why this change?

Reply:
need the id to be available so we know which one was reported